build-parent/pom.xml

```diff
@@ -161,9 +161,6 @@
 
         <!-- OpenLineage java -->
         <version.open.lineage.java>1.23.0</version.open.lineage.java>
-
-        <!-- Used for pinning version specifically test-mda-model-->
-        <version.test.resteasy>3.15.6.Final</version.test.resteasy>
```

Comment:
I: No longer needed, as this was just a workaround for the version mismatch issue in the test-mda modules.